core/src/error/kind.rs

```diff
@@ -130,7 +130,7 @@ impl ErrorUnknownField {
     }
 
     #[cfg(feature = "diagnostics")]
-    pub fn to_diagnostic(self, span: Option<::proc_macro2::Span>) -> ::proc_macro::Diagnostic {
+    pub fn into_diagnostic(self, span: Option<::proc_macro2::Span>) -> ::proc_macro::Diagnostic {
```

Comment:
Can you please add this to `CHANGELOG.md` under an "Unreleased" header at the top? This is technically a breaking change.

Reply:
Oh is it? I thought this method is not public because the containing struct is marked `pub(in crate::error)` and the function is not shown [in the online docs](https://docs.rs/darling/0.12.4/darling/index.html?search=to_diagnostic)

Reply:
Right you are.